Start at a "getting started" page in the intro section. (#10)
quorumcontrol/cryptocolosseum-docs: docs/firstbet.md

---
title: Making your fist bet in the Crypto Colosseum
sidebar_label: Getting Started
slug: '/firstbet'
---

Crypto Colosseum is (in the beginning) a simple gambling game.

Gladiators have tournaments.  You place your bet on which of the four gladiators will win the tournament.  You get a share of the prize purse if you are right.

To get started the first thing you need to do is sign up.  Click on the orange "Sign in / Sign up" in the upper right hand corner to do so. Its easy we promise.

![image](signin.png)

> We have provided several different ways for you to do so but if you are new to the game and to crypto, the easiest way is to use an existing account such as gmail.  When you do this we are actually creating a special vault or wallet for you to recieve and spend your crypto currency (PTG) from.  (As well as other things.)  You own it through your gmail or social media account we can not access it without your permission.

Reviewer: ```suggestion
There are several different ways to login, but if you are new to the game and to crypto, the easiest way is to use an existing account (like gmail).  When you do this we are actually creating a special vault or wallet to receive and spend your crypto currency (PTG).  (As well as hold your in-game NFTs)  You own it and we can not access it without your permission.
```

![image](picktraditional.png)

> The creation of your wallet and the linking to your existing account elsewhere is done through Torus.  You don't need to worry about that for now but you can access that through the "T" in the bottom left at all times.

Reviewer: link to torus?

Author: Done.

![image](lobby.png)

We constantly post links to upcoming tournaments in [our discord](https://discord.gg/Z2S3EtQKCn) so you should join us there to hear about upcoming tournaments, contests and special events. 

Now the last step is to place a bet on the tournament page.  There is a lot on the tournament page but all you need to worry about is which gladiator you want to bet on.  Pick one and find the associated orange "Bet Now" button and click it.  Below you can see what some of the other info is but for your first bet, bet small (maybe 100 PTG) and pick whichever appeals to you the most.

![image](tournament.png)

If you are the type to want to know more before you bet check out [combat basiscs](https://docs.cryptocolosseum.com/gamemechanics/combatexample) to learn more.

Once you click to place your bet a popup with ask you how much to bet.  Then once you submit that your wallet will check if you are really sure you want to put some of the PTG in your pocket on the line.  Confirm that and you are off and running!  (Look for popup blockers etc if you dont see the confirmation requests).

> The game holds surprising complexity and there are many aspects to learn but bet a few times and then explore the item market (on bonding curves), bidding on a gladiator contract (all income generating NFTs) and even explore getting your own share of the factions and their winnings.

Reviewer: ```suggestion
The game holds surprising complexity and there are many aspects to learn, but bet a few times and then explore the item market (on bonding curves), bidding on a gladiator contract (income generating NFTs) and even explore getting your own share of the factions and their winnings.
```

Most of all have fun!  

> Its whimsical violence.  Its degengerate gambling and we just gave you free money.  

Reviewer: ```suggestion
It's whimsical violence.  It's degengerate gambling and The Arena just gave you free money.  
```

> Party time!

Reviewer: ```suggestion
Battle time!
```
